include/model.php

```diff
@@ -186,6 +187,11 @@ public function get_languages_list( $args = array() ) {
 			}
 		}
 
+		// Remove default language if requested.
+		if ( ! empty( $args['hide_default'] ) ) {
+			$languages = wp_list_filter( $languages, array( 'is_default', true ) );
+		}
```

Comment:
Just a suggestion to merge both filters:
```php
		$languages = array_filter(
			$languages,
			function( $lang ) use ( $args ) {
				return ! ( ( $args['hide_empty'] && $lang->get_tax_prop( 'language', 'count' ) ) || ( $args['hide_default'] && $lang->is_default ) );
			}
		);

Reply:
I like it, done in 0ed8f3bc. I've just used variables to make it clearer and checked array keys exist.

Reply:
Instead of using `isset()`, should we use `wp_parse_args()`?